src/lexer.jl

```diff
@@ -637,20 +637,20 @@ end
 
 # Parse a token starting with a quote.
 # A '"' has been consumed
-function lex_quote(l::Lexer)
```

Comment:
Good point, I'll have it return an empty token. Related to this, is it worthwhile defining a `const emptytoken = Token()` for cases such as this (I have other instances in my parser) ?